libs/common/setup.go

```diff
@@ -49,7 +51,12 @@ func Setup(serviceName, version string, auth bool) {
 	}
 
 	if auth {
-		setupKeycloak()
+		if strings.ToLower(hwutil.GetEnvOr("FAKE_TOKEN_ENABLE", "false")) == "true" && Mode == DevelopmentMode {
+			FakeTokenEnable = true
+			log.Warn().Msg("FAKE_TOKEN_ENABLE is set to true, accepting fake tokens")
```

Comment:
I think this should be an error-level log. If we see this in prod, it's an error and noticed faster. In dev environments, the devs know it can be ignored.